example-a.nut

```diff
@@ -1,31 +1,33 @@
 #require "APDS9007.class.nut:2.0.0"
 
 // value of load resistor on ALS
-const RLOAD = 47000.0
+const RLOAD = 47000.0;
 
-// use pin#5 as analog in
-analogInputPin <- hardware.pin5
-analogInputPin.configure(ANALOG_IN)
+// use pin#5 as analog input
+analogInputPin <- hardware.pin5;
+analogInputPin.configure(ANALOG_IN);
 
-// use pin#7 as digital out
-enablePin <- hardware.pin7
-enablePin.configure(DIGITAL_OUT, 0)
+// use pin#7 as enable poin
+enablePin <- hardware.pin7;
+enablePin.configure(DIGITAL_OUT, 0);
 
 // initialize driver class
-lightsensor <- APDS9007(analogInputPin, RLOAD, enablePin)
+lightsensor <- APDS9007(analogInputPin, 47000 , enablePin);
 
 // enable sensor
 lightsensor.enable(true);
 
 // get readout
-readLightLevel <- @() lightsensor.read(function(result) {
-    if ("err" in result) {
-        server.log("Error Reading APDS9007: " + result.err);
-        return;
-    }
-    server.log("Light level = " + result.brightness + " Lux");
-    imp.wakeup(2, readLightLevel); // repeat in 2 seconds
-});
+function readLightLevel() {
+    lightsensor.read(function (result) {
+        if ("err" in result) {
+            server.log("Error Reading APDS9007: " + result.err);
+            return;
+        }
+        server.log("Light level = " + result.brightness + " Lux");
+        imp.wakeup(2, readLightLevel); // repeat in 2 seconds
+    });
+};
 
-// start reading light level every 2 seconds
+// start reading light level every 2 seconds, the first reading will arrive in 5 secs
 readLightLevel();
```

Comment:
This will throw an error for the first five seconds. If this is intentional, then document it. If it is an error, then correct it so the first `readLightLevel()` call takes place after 5.0 seconds.